client/capstone-app/src/main.jsx

```diff
@@ -14,6 +16,8 @@ const router = createBrowserRouter(
                   <Route path='/' element={<App />}>
                     <Route path='signup' element={<SignUp />}/>
                     <Route path='login' element={<Login />}/>
+                    <Route path='get-started' element={<GetStarted />}/>
+                    <Route path='map/:currentCoordinates/:destinationCoordinates' element={<MapPage />}/>
```

Comment:
I would refine how you represent the lat and long parameters in your URL. For instance, you often need to [escape values within your URL](https://www.werockyourweb.com/url-escape-characters/) otherwise the URL may break as a hyperlink, in an app, or otherwise. This is bc certain characters represent something special in HTTP/URL convention like '&' --> '%26'

At minimum always consider calling a URL escape library before jamming the parameter into your router. Even better, you should understand best practices with that type of value with respect to URL-parameterization.

This could be illustrative of best practices on safely representing lat and longitude parameter in URL
https://stackoverflow.com/questions/2660201/what-parameters-should-i-use-in-a-google-maps-url-to-go-to-a-lat-lon

Reply:
Addressed in [PR #6](https://github.com/desolagiwa/metaU-capstone/pull/6)